web/controllers/survey_controller.ex

```diff
@@ -41,9 +41,9 @@ defmodule Ask.SurveyController do
     render(conn, "index.json", surveys: surveys)
   end
 
-  def create(conn, params = %{"folder_id" => folder_id}), do: create(conn, params, folder_id)
+  def create(conn, params = %{"project_id" => project_id}) do
```

Comment:
This is a minor code improvement. It comes from [this commit](https://github.com/instedd/surveda/commit/d9cce3b425ccca2f38be091f68f3d9e819f11eae). It removes an unneeded function overload.